index.ts

```diff
@@ -0,0 +1,28 @@
+import {install} from 'source-map-support';
+install();
+
+import {AndroidDriver} from './lib/driver';
+
+export type * from './lib/commands/types';
+export {ANDROID_DRIVER_CONSTRAINTS as commonCapConstraints} from './lib/constraints';
+export * from './lib/driver';
+export {SETTINGS_HELPER_PKG_ID, default as androidHelpers} from './lib/helpers/android';
+export type * from './lib/helpers/types';
+export {
+  CHROMIUM_WIN,
+  NATIVE_WIN,
+  WEBVIEW_BASE,
+  WEBVIEW_WIN,
+  default as webviewHelpers,
+} from './lib/helpers/webview';
+
+/**
+ * @privateRemarks `androidCommands` was previously exported as an object
```

Comment:
we only need mixin commands there. Methods defined in the driver class itself must be ignored

Reply:
why?

Reply:
because AndroidDriver class itself contains stuff which is specific for the obsolete driver. It must not be used neither in uia2 nor in espresso

Reply:
ok, I changed this to the way it was

Reply:
but also, ideally, uia2 and espresso would be subclasses of `AndroidDriver`, which renders it moot; they will get everything in `AndroidDriver` and would need to override any prop on the prototype they don't want

Reply:
right now it is a collection of various mixins and helper methods which matter in this component. 
UIA2 and Espresso are too different to prefer subsclassing over composition.